src/clarity-angular/emphasis/alert/_alert.clarity.scss

```diff
@@ -126,11 +64,11 @@ $clr-alert-colors: (
     //close alert button.
     .alert-item > span,
     .alert-text {
-        display: inline-block;
+        //display: inline-block; // TODO get buy in on removal during PR
         flex-grow: 1;
         flex-shrink: 1;
         flex-basis: 98%; //needed for IE11
-        max-width: 98%;
+        //max-width: 98%; // TODO get buy in on removal during PR
```

Comment:
This is needed for IE11. Why are we commenting this out?

Reply:
I thought it only flex-basis was required. Will uncomment.

Reply:
Added a comment for the attrs req'ed for IE11 and uncommented this.